### ISSUE TITLE
`Lint/UnusedMethodArgument` false positive when argument used in a Hash without curly brackets

### ISSUE THREAD

**Issue by stanley90** [rubocop/rubocop#14609]
## Steps to reproduce the problem

```
def get_index(params = {})
  get :index, format: :json, params:
```

## Expected behavior

Should not report an unused method argument.

## Actual behavior

Reports `Lint/UnusedMethodArgument: Unused method argument - params`
Note: If the call is `get :index, { format: :json, params: }`, there is no offense.

## RuboCop version

```
1.81.1 (using Parser 3.3.9.0, Prism 1.5.1, rubocop-ast 1.47.1, analyzing as Ruby 3.4, running on ruby 3.4.6) [arm64-darwin25]
  - rubocop-rails 2.33.4
  - rubocop-performance 1.26.0
```

**Comment by stanley90** [comment]
My bad, the syntax is not valid
